src/pages/index.js

```diff
@@ -50,9 +50,24 @@ function HomepageHeader() {
     <div className="content-block home-intro">
       <div className="block-content text-centered">
         <h1>
-          <strong>The package manager for Kubernetes</strong>
+          <strong>
+            <Translate
+              id="homepage.title"
+              description="The main title on the homepage">
+              The package manager for Kubernetes
+            </Translate>
+          </strong>
```

Comment:
^ general format is to add the <Translate> tag, assign an id and description, and then when you run write-translations, these will get pulled into the code.json file